backend/internal/repository/blueprint.go

```
@@ -0,0 +1,264 @@
+	}
+	return bps, nil
+}
+
+func (r *blueprintRepository) CreateBlueprint(ctx context.Context, b *blueprint.Blueprint, components []*blueprint.BlueprintComponent) (err error) {
+	tx, err := r.db.BeginTxx(ctx, nil)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		if p := recover(); p != nil {
+			tx.Rollback()
```

golangci-lint (1.64.8)

Error return value of `tx.Rollback` is not checked

(errcheck)